cert-manager/cert-manager/base/service-accounts.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
nit - even with multiple resources most manifests have just called it service-account.yaml

Reply:
I'll remove all 's' trailing chars from the resources.